content/docs/next-release/operator.md

```diff
@@ -0,0 +1,645 @@
+
+The agent can be injected as a sidecar on the instrumented application or as a daemonset.
+
+The query and collector services are configured with a supported storage type - currently Cassandra or Elasticsearch. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
+The main additional requirement is to provide the details of the storage type and options, for example:
+
+```yaml
+    storage:
+      type: elasticsearch
+      options:
+        es:
+          server-urls: http://elasticsearch:9200
+```
+
+
+## Streaming strategy
+
+The `streaming` strategy is designed to augment the `production` strategy by providing a streaming capability that effectively sits between the collector and the backend storage (Cassandra or Elasticsearch). This provides the benefit of reducing the pressure on the backend storage, under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (Kafka).
+
+The only additional information required is to provide the details for accessing the Kafka platform, which is configured in the `ingester` component:
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-streaming
+spec:
+  strategy: streaming
+  collector:
+    options:
+      kafka: # <1>
+        producer:
+          topic: jaeger-spans
+          brokers: my-cluster-kafka-brokers.kafka:9092
+  ingester:
+    options:
+      kafka: # <1>
+        consumer:
+          topic: jaeger-spans
+          brokers: my-cluster-kafka-brokers.kafka:9092
+      ingester:
+        deadlockInterval: 0 # <2>
+  storage:
+    type: elasticsearch
+    options:
+      es:
+        server-urls: http://elasticsearch:9200
+```
+<1> Identifies the Kafka configuration used by the collector, to produce the messages, and the ingester to consume the messages.
+
+<2> The deadlock interval can be disabled to avoid the ingester being terminated when no messages arrive within the default 1 minute period
+
+TIP: A Kafka environment can be configured using [Strimzi's Kafka operator](https://strimzi.io/).
+
+
+# Creating a simple Jaeger instance
+
+You can view example custom resources for different Jaeger configurations [on GitHub](https://github.com/jaegertracing/jaeger-operator/tree/master/deploy/examples).
+
+The simplest possible way to install is by creating a YAML file like the following:
+<!-- TODO - Provide an overview of what we’re getting by default: one container, running what services?  Just the server that serves up the console?  Or is this the operator version of the All in one Executable?  Which installs the Jaeger UI, collector, query, and agent, with an in memory storage component?-->
+
+.simplest.yaml
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simplest
+```
+
+The YAML file can then be used with `kubectl`:
+<!-- TODO - Add OKD commands and tabs shortcode. -->
```

Comment:
tabs shortcode?

Reply:
Someone asked if we could add tabs for the different environments the way the Istio docs have them (https://istio.io/docs/setup/kubernetes/install/kubernetes/#installation-steps for an example)  and it looks like the Istio Hugo site uses a special shortcode to generate the tabs.